source/develop/release-management/features/engine/metrics-store.html.md

```diff
@@ -53,6 +53,105 @@ Once you have finished this step, you should have:
 
   * Kibana - <https://kibana.{hostname}>
   * OpenShift portal - <https://openshift.{hostname}>
+  
+  
+
+**Update mux pod resources**
+
+1. ssh into the logging machine and use the command line.  
+**Note:** You can probably do all of this with the OpenShift UI too - https://logging-machine:8443
+
+2. Run
+
+       # $ oc edit dc logging-mux
+
+Edit the section
+
+      spec:
+        template:
+          spec:
+            containers:
+      ...
+              resources:
+                limits:
+                  cpu: 500m
+                  memory: 2Gi
+
+   For cpu, units are in millicores.  500m means 0.5 cores.
+
+   If you want mux to use more cpu, you can update it up to 1000m.
+
+   In case mux is maxed out of cpu, and you have already increased the mux cpu to 1000m, you should scale up an additional pod.
+
+   For memory, 2Gi means 2 Gigabytes.  Increase as needed.
+
+   If you change and save the new config, that should automatically trigger a redeployment of all mux pods.
+   
+
+
+3. Run:
+
+       # oc get pods -l component=mux
+
+This should show that mux recently restarted.  If not, use
+
+       # oc rollout latest dc/logging-mux
+
+and
```

Comment:
and - the first command is to start the rollout, the second is to follow `-w` it until it is completed